### PR TITLE
Add procurement principles

### LINKED ISSUE #372: Devise and publish a procurement policy for the Foundation
Steps:
- [ ] find out what Dutch law requires (eg, what paper trail do we need for tax purposes?) (See also the [VNG model procurement policy](https://vng.nl/onderwerpenindex/recht/aanbesteden/nieuws/geactualiseerd-model-inkoop-en-aanbestedingsbeleid), which is apparently widely used)
- [ ] work with team to identify our values when it comes to procurement (eg, do we care about the provider being non-discriminatory, environmentally sustainable or a local business?)
- [ ] look at the values/constraints of the procurement policies of potential members so we're relatively consistent (at least in values, even if we don't have the same legal constraints)
- [ ] create a draft (for example, this is a [quite staid English-language template](https://www.fundsforngos.org/ngo-operational-policies/4-procurement-policy-for-ngos/))

### PR DESCRIPTION
**Pull request by demkodo**
Took a stab at describing steps we can take in procurement process, open for feedback and suggestions. Thank you

closes #372

-----
[View rendered activities/procurement/principles.md](https://github.com/publiccodenet/about/blob/add-procurement-process/activities/procurement/principles.md)

### CODE REVIEW

File: activities/procurement/principles.md [hunk 1]
```diff
@@ -14,3 +14,12 @@ This means the services and products we use need to be:
 * sustainable for the organization as well as society and the planet
 
 The principles of our procurement build on and extend our [cultural values](../../organization/cultural-values.md) and [communication principles](../communication/communication-principles.md).
+
+## Procurement process
+
+To make sure the procurement process is fair, we adhere to the following requirements:
+
+* we request 3 - 4 quotes from different providers
```
Comment (bvhme) [inline activities/procurement/principles.md:22]: I'd keep this at 'a proposal' from one party, unless we hit the European limits, at which moment we can go with a more open procurement.

File: activities/procurement/principles.md [hunk 2]
```diff
@@ -14,3 +14,12 @@ This means the services and products we use need to be:
...
+* check if prices are competitive and in line with the market
```
Comment (bvhme) [inline activities/procurement/principles.md:23]: I'd suggest judging them to price and value in adition to market conformity.

File: activities/procurement/principles.md [hunk 3]
```diff
@@ -14,3 +14,12 @@ This means the services and products we use need to be:
...
+* if products are sustainable and / or local
```
Comment (bvhme) [inline activities/procurement/principles.md:24]: Here I would check if it is in line with our 'procurement principles' in `principles.md`. Add a link :link:

File: activities/procurement/principles.md [hunk 4]
```diff
@@ -14,3 +14,12 @@ This means the services and products we use need to be:
...
+* final decision taken with operations team
```
Comment (bvhme) [inline activities/procurement/principles.md:25]: The order in this list is important so it should be an ordered list.

Ordered lists have numbers in front of them, either incrementing or all the same. I recommend replacing `* ` with `1. `.

Comment (bvhme) [inline activities/procurement/principles.md:25]: I would replace this with something like, 'the order is placed by the operations team and signed off by the chief executive and board of directors if needed'.

File: activities/procurement/principles.md [hunk 5]
```diff
@@ -14,3 +14,12 @@ This means the services and products we use need to be:
 * sustainable for the organization as well as society and the planet
 
 The principles of our procurement build on and extend our [cultural values](../../organization/cultural-values.md) and [communication principles](../communication/communication-principles.md).
+
+## Procurement process
+
+To make sure the procurement process is fair, we adhere to the following requirements:
```
Comment (bvhme) [inline activities/procurement/principles.md:20]:
```suggestion
Our process for procurement of services, and in some cases goods, is as follows:
```

File: activities/procurement/principles.md (hunk 4 above)
Comment (bvhme) [reply activities/procurement/principles.md]: This was meant as a comment on the last point only

File: activities/procurement/index.md [hunk 6]
```diff
@@ -9,3 +9,7 @@ To fulfill our mission, we need to get things and services from others.
 How we procure:
 
 * [Procurement principles](principles.md)
+
+Procurement process:
+
+* [Procurement process](process.md)
```
Comment (bvhme) [inline activities/procurement/index.md:15]: I think these two can be in the same list, I'd argue 'procurement process' still falls in 'how we procure'. If you don't think so feel free to come up with a better name for this list.

```suggestion
* [Procurement process](process.md)
```

File: activities/procurement/principles.md [hunk 7]
```diff
@@ -13,4 +13,6 @@ This means the services and products we use need to be:
 * replaceable
 * sustainable for the organization as well as society and the planet
 
-The principles of our procurement build on and extend our [cultural values](../../organization/cultural-values.md) and [communication principles](../communication/communication-principles.md).
+The principles of our procurement build on and extend our [cultural values](../../organization/cultural-values.md) and [communication principles](../communication/communication-principles.md). 
+
+Read more on our [procurement process.](process.md)
```
Comment (ElenaFdR) [inline activities/procurement/principles.md:18]:
```suggestion
Read more on [our procurement process](process.md).
```

File: activities/procurement/principles.md [hunk 8]
```diff
@@ -13,4 +13,6 @@ This means the services and products we use need to be:
 * replaceable
 * sustainable for the organization as well as society and the planet
 
-The principles of our procurement build on and extend our [cultural values](../../organization/cultural-values.md) and [communication principles](../communication/communication-principles.md).
+The principles of our procurement build on and extend our [cultural values](../../organization/cultural-values.md) and [communication principles](../communication/communication-principles.md).
```
Comment (bvhme) [inline activities/procurement/principles.md:16]: Trailing spaces :)

```suggestion
The principles of our procurement build on and extend our [cultural values](../../organization/cultural-values.md) and [communication principles](../communication/communication-principles.md).
```

File: activities/procurement/process.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+---
+
+# Procurement process
+
+Our process for procurement of services, and in some cases goods, is as follows:
+
+1. we request a proposal or quote
+2. we check if price and value are in line with the market
+3. if products and services are in line with our [procurement principles](principles.md)
+4. the order is placed by the operations team and signed off by the chief executive and board of directors, if needed
```
Comment (ElenaFdR) [inline activities/procurement/process.md:12]: I have some style edit suggestions.

Numbered lists are usually full sentences and don't require lead-in sentences. The active tense is usually easier to read than the passive.

Proposal:

This is how we procure all services and some goods.

1. We request a proposal or quote.
2. We check if price and value are in line with the market.
3. We check that the proposed procurement meets our [procurement principles](principles.md).
4. The operations team places the order, after approval by the chief executive and board of directors, if needed.

**Comment by bvhme** [comment]
Hey @demkodo, let me know if you need any help with getting this pull request in?

**Comment by demkodo** [comment]
Hi B, not using Mattermost yet but great to see notifications from Mattermost concerning GitHub <3
This is ready and Elena said she would review and close. But, please feel free to do that on her behalf, also handy since it's mostly your feedback I incorporated. Merci

**Comment by bvhme** [comment]
Good to hear, both @ElenaFdR and I have reviewed here and left some further suggestions. For you to incorporate or not.

**Comment by bvhme** [comment]
@demkodo Elena's review ended with her 'requesting changes', this means when you're done with Elena's suggestions you can re-request her review. And when it passes that and the Travis check it can be merged.

**Comment by demkodo** [comment]
Got it. I strangely did not receive a notification, only now this one from yesterday via Mattermost, which is a great sign that that at least works :) Will update accordingly.

File: activities/procurement/process.md (hunk 9 above)
Comment (demkodo) [reply activities/procurement/process.md:12]: Agreed :)

Comment (ElenaFdR) [reply activities/procurement/process.md:12]: Hey hey! These changes have not been applied to the file.

Comment (demkodo) [reply activities/procurement/process.md:12]: Great (upside down smiley) -- on it

Comment (demkodo) [reply activities/procurement/process.md]: Done

File: activities/procurement/process.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+type: Resource
+---
+
+# Procurement process
+
+This is how we procure all services and some goods.
+
+1. We request a proposal or quote.
+2. We check if price and value are in line with the market.
+3. We check that the proposed procurement meets our [procurement principles](principles.md)
```
Comment (bvhme) [inline activities/procurement/process.md:11]:
```suggestion
3. We check that the proposed procurement meets our [procurement principles](principles.md).
```

File: activities/procurement/process.md [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+4. The operations team places the order, after approval by the chief executive and board of directors, if needed.
```
Comment (bvhme) [inline activities/procurement/process.md:12]:
```suggestion
4. The [operations team](../../organization/staff.md#operations) places the order, after approval by the chief executive and board of directors, if needed.
```

**Comment by demkodo** [comment]
> In the past we've gone about fixing this in this issue: #372
> 
> Is there something we want to take from there? I guess this issue closes #372.

I remember looking at the VNG procurement policy and thought it would be a great script for a multi-season drama series on procurement, shuddered and never looked at it again :) But, yes, all the other notes have been taken into account and thus I believe we can close #372 at least for now (until we are prompted by e.g. BZK to expand our policy).

**Pull request merged**